lua-docs/content/reference/Player.md

```diff
@@ -5,27 +5,115 @@ keywords: particubes, game, mobile, scripting, cube, voxel, world
 
 # Player
 
-The Player type describe a human player, playing the game as well as its ingame character/avatar.
+Object representing a player, someone who's connected to the game.
 
 - Caps fields are read-only (expect for exceptions)
 - Lowercase fields are free to use (read/write)
 
 ## Fields
 
-### Id (Integer)
+### Id (Integer, read-only)
 
-### IsLocal (Boolean)
+Unique player id for played game. A different id can be attributed after reconnection.
+
+### Username (String, read-only)
+
+`Player`'s account username.
+
+### IsLocal (Boolean, read-only)
+
+When the script runs locally (on player's device), `Player.IsLocal` is true for local player only.
 
 ### IsOnGround (Boolean)
 
-### Jump (Number3)
+`Player.IsOnground` is true if there's a block underneath `Player`'s feet.
+
+### Jump (Function)
+
+`Player.Jump` is triggered when jumping. `nil` by default, it has to be defined for players to jump! 
+
+Since the `Player.Function` is triggered by each player locally, it only only makes sense to define it for the local player.
```

Comment:
did you mean `Player.Jump` instead of `Player.Function` ?

Reply:
Fixed!